src/newcli.ml

```diff
@@ -253,7 +263,12 @@ let open_channels () =
     try_lwt
```

Comment:
This is the old camlp4 syntax... this makes me wonder if this package is maintained at all. Does it even compile in the new xs-opam?